docs/registry/create/_index.md

```diff
@@ -437,7 +437,7 @@ func (b *myBase) Reconfigure(ctx context.Context, deps resource.Dependencies, co
 
     geometries, err := kinematicbase.CollisionGeometry(conf.Frame)
     if err != nil {
-        b.logger.Warnf("base %v %s", b.Name(), err.Error())
+        b.logger.CWarnf(ctx, "base %v %s", b.Name(), err.Error())
```

Comment:
It was a decision the NetCode team made a while ago. Most loggers have methods that look like `Debug`, `Debugf`, and `Debugw` for all log levels. Most loggers also do not take a `context.Context`. We wanted to take a context on our logging methods to pass some "extra" information along with each log that could help with debugging the path a single operation was taking through the RDK. When an alternative version of a method that is otherwise identical to the original but offers a `context.Context` param is written in Golang, it's common practice to name that method either `FooCtx` of `CFoo`. We went with the latter.